TypeCobol.LanguageServer/Completion Factory/CompletionFactory.cs

```diff
@@ -610,12 +610,16 @@ public static IEnumerable<CompletionItem> GetCompletionForTo(FileCompiler fileCo
             {
                 if (seekedDataTypes.Count == 0) //If a Datatype hasn't be found yet. 
                 {
-                    var foundedVars =
-                        node.SymbolTable.GetVariablesExplicit(
-                            new URI(qualifiedNameTokens.Select(t => t.Text)));
+                    var foundedVar = node.SymbolTable.GetVariablesExplicit(new URI(qualifiedNameTokens.Select(t => t.Text))).FirstOrDefault();
 
-                    if (foundedVars != null && foundedVars.Any())
-                        seekedDataTypes.Add(foundedVars.First().DataType);
+                    if (foundedVar != null)
+                    {
+                        seekedDataTypes.Add(foundedVar.DataType);
+                        if (foundedVar.TypeDefinition?.PrimitiveDataType != null)
+                        {
+                            seekedDataTypes.Add(foundedVar.TypeDefinition.PrimitiveDataType);
```

Comment:
You can always add :
`seekedDataTypes.Add(DataType.Alphanumeric);`
Every type can be moved in an alphanumeric

Reply:
Done.